### PR TITLE
Add e2e jobs for Chrome and firefox browsers 

### PR DESCRIPTION
**Pull request by tendomart**
# Pull Requests Requirements

## Summary
Add e2e jobs for Chrome and firefox browsers 

These run the default inbuilt cypress browser, chrome and Firefox.

Should be triggered on push

https://github.com/tendomart/OpenELIS-Global-2/commit/2251652afe9df022a6d317e2f81e94a1d5e581f6

Hi @mozzy11  should focus on this commit 
https://github.com/tendomart/OpenELIS-Global-2/commit/2251652afe9df022a6d317e2f81e94a1d5e581f6

### CODE REVIEW

File: frontend/src/components/admin/Admin.js
Comment (Ankit-Matth) [inline frontend/src/components/admin/Admin.js:1]: Are changes to this file necessary for adding end-to-end (e2e) jobs for Chrome and Firefox browsers? I think these came by mistake from a previous commit.

File: frontend/src/components/common/GenericBreadCrumb.js
Comment (Ankit-Matth) [inline frontend/src/components/common/GenericBreadCrumb.js:1]: Are changes to this file necessary for adding end-to-end (e2e) jobs for Chrome and Firefox browsers? I think these came by mistake from a previous commit.

File: frontend/src/components/validation/Index.js
Comment (Ankit-Matth) [inline frontend/src/components/validation/Index.js:1]: Are changes to this file necessary for adding end-to-end (e2e) jobs for Chrome and Firefox browsers? I think these came by mistake from a previous commit.

**Review by Ankit-Matth — CHANGES_REQUESTED** [review]
Hey @tendomart, I believe some unnecessary changes were accidentally included in this PR from previous commits. Could you please take a look at it?

**Review by mozzy11 — CHANGES_REQUESTED** [review]
@tendomart Thanks for working on this. 
I think you have changes that are not needed in this PR

**Comment by tendomart** [comment]
Yes thanks, @mozzy11 , I thought you would selectively pick the last commit. but will remove them sometime today.

**Comment by mozzy11** [comment]
THanks @tendomart can you adress the CI Build failre and merge conflicts ?

**Comment by tendomart** [comment]
Alright @mozzy11  will find some time later today and look into resolving this

**Pull request closed without merging**